### DISCUSSION TITLE
Solspace Calendar as Element Connections

### DISCUSSION THREAD

**Discussion by danystad** [solspace/craft-freeform#74] [Feature Requests]
I'm looking for a way to create a public event submission form which creates a Calendar event as a draft ( disabled entry) and with email notifications using Freeform, so that a website administrator can review the event submission and publish it.

I realize there is a documented way to create Calendar events on the front end, but there isn't an easy way to set up email notifications and disable submitted entries which are both necessary for reviewing any publicly submitted events.

I would like to submit this as a feature request for the Element Connections, to have it connect with element types from your Calendar plugin. 

Having the Solspace Calendar and Freeform plugins work together would be a great enhancement and selling feature.

**Comment by kjmartens** [comment]
Thanks for your feedback, @danystad. This is currently under consideration for a future release. :)

**Comment by kjmartens** [comment]
This has been added in Freeform 3.11. :)

**Comment by TUSAB-web** [comment]
This feature looks very useful. Might I suggest that another integration be considered? 

Currently, in freeform, one can create a “select” field and then populate the options from Craft Elements. Solspace Calendar events are not available in the list of sections one can add though. 

Our use case: We perform concerts which are added as Solspace Calendar entries, and have a Freeform feedback form on the website. It would be great if people replying to the survey could select the concert they attended from a dropdown. It doesn’t appear possible at this time, although I could be missing something.

**Reply by kjmartens** [reply]
Thanks for the suggestion, @TUSAB-web. Can you create a new Feature Request for this, and we'll revisit this and see if we can add the feature in the not-too-distant future. 🙂
